src/main/resources/openshift/deploy.sh

```diff
@@ -39,18 +39,32 @@ rc=$?; if [[ $rc != 0 ]]; then echo "Missing deployment. Please build and copy r
 echo
 echo "Openshift project"
 echo "  -> Create Openshift project (${OCP_PROJECT})"
-oc new-project ${OCP_PROJECT} 2>/dev/null > /dev/null
+oc new-project ${OCP_PROJECT} > /dev/null
 sleep 1
-
-oc policy add-role-to-user view system:serviceaccount:$(oc project -q):eap-service-account -n $(oc project -q)
-oc policy add-role-to-user view system:serviceaccount:$(oc project -q):sso-service-account -n $(oc project -q)
-
 echo "  -> Switch to project"
-oc project ${OCP_PROJECT}  2>&1 > /dev/null
+oc project ${OCP_PROJECT} > /dev/null
+sleep 1
+echo "  -> Register service accounts"
+oc policy add-role-to-user view system:serviceaccount:${OCP_PROJECT}:eap-service-account -n ${OCP_PROJECT}
+oc policy add-role-to-user view system:serviceaccount:${OCP_PROJECT}:sso-service-account -n ${OCP_PROJECT}
+sleep 1
+echo "  -> Verify ImageStreams"
+sleep 1
+M=0
+until [ ${M} -ge 50 ]
```

Comment:
Here the idea is to check if the required middleware image streams are already available in the used OCP instance. While testing / deploying the Web Console on OCP, I had a couple of time the following issue:

> The ImageStreamTag "redhat-sso70-openshift:latest" is invalid: from: Error resolving ImageStreamTag redhat-sso70-openshift:latest in namespace openshift: unable to find latest tagged image

This is a fix for it. Of course, it would be a good idea to exit with a failure error code if the image streams are not available after 50 attempts.